tic-tac-toe-app/styles.css

```diff
@@ -0,0 +1,21 @@
+.box{
```

Comment:
Each of your styles aligns to a className in your app. Solid. 💯

Reply:
Thank you Keyairra!